website/project/model.py

```diff
@@ -179,6 +179,32 @@ class MetaData(GuidStoredObject):
     date_modified = fields.DateTimeField(auto_now=datetime.datetime.utcnow)
 
 
+def validate_contributor(guid, contributors):
+    user = User.find(
+        Q('_id', 'eq', guid) &
+        Q('is_claimed', 'eq', True)
+    )
+    if user.count() != 1:
+        raise ValidationValueError('User does not exist or is not active.')
+    elif guid not in contributors:
+        raise ValidationValueError('Mentioned user is not a contributor.')
+    return True
+
+def get_valid_mentioned_users_guids(comment, contributors):
+    """ Get a list of valid users that are mentioned in the comment content.
+
+    :param Node comment: Node that has content and ever_mentioned
+    :param list contributors: List of contributors on the node
+    :return list new_mentions: List of valid users mentioned in the comment content
+    """
+    new_mentions = set(re.findall(r"\[[@|\+].*?\]\(\/([a-z\d]{5})\/\)", comment.content))
```

Comment:
Ah ok. My mistake. If we're going to reuse this regex, we should probably make it a constant